src/quo2/components/inputs/input/view.cljs

```diff
@@ -135,7 +135,7 @@
        (assoc props
               :accessibility-label :password-input
               :auto-capitalize     :none
-              :auto-complete       :new-password
+              :auto-complete       :password
```

Comment:
Is this needed for the purpose of this PR?

Reply:
I got an error when running the builds on my android device so thought of fixing it here.
This could be done in a separate PR as well :)

Reply:
If your error has its roots in the RN version upgrade I wouldd keep this change, otherwise I suggest to to it on a separate PR. If we have to rollback for some reason it would affect this change, but not if you do it separately.

Reply:
funny how this prop ever worked for `text-input` component, while we are in `react-native` 0.63.3 because the auto-complete prop was introduced in `react-native` 0.66 : https://reactnative.dev/docs/0.66/textinput#autocomplete-android
It is an `Android` only prop and that explains why I got the error on `Android`.

Could not find that prop for `text-input` in documentation of 0.63 : https://reactnative.dev/docs/0.63/textinput

Also it should have been `:password-new` instead of `:new-password` :)

Reply:
fixed here: https://github.com/status-im/status-mobile/pull/15486/commits/f081b191c13bed1ef371955efa7c821bc2cbb6f4